cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SecurityPrivilegesTest.scala

```diff
@@ -156,6 +156,55 @@ class SecurityPrivilegesTest extends DocumentingTest with QueryStatisticsTestSup
         })) {
           resultTable()
         }
+
+        p("Available privileges can also be output as Cypher commands with the optional `AS COMMAND[S]`.")
```

Comment:
Nice addition! One thing I was thinking about it that all subsections here start with describing `SHOW xxx PRIVILEGE` and then `SHOW xxx PRIVILEGE AS COMMANDS`. Maybe we should consider to flip the order if we want users to use `AS COMMANDS`? Because I believe that people in general might be lazy and stop reading when they found the first solution to their problem...

Reply:
Might be a good idea (but will need more rewriting of text)